projects/rulenode-core-config/src/lib/components/common/msg-metadata-chip.component.html

```diff
@@ -2,7 +2,9 @@
   <div class="fixed-title-width">{{ labelText }}</div>
   <mat-chip-listbox formControlName="chipControl">
     <mat-chip-option *ngFor="let option of selectOptions"
-                     selectable="{{ chipControlGroup.get('chipControl').value !== option.value }}" color="primary"
+                     selectable="{{ chipControlGroup.get('chipControl').value !== option.value }}"
+                     color="primary"
+                     [selected]="chipControlGroup.get('chipControl').value === option.value"
```

Comment:
Why is it written selected if this value should be taken from **formControlName**

Reply:
Done